.github/workflows/kubeconform-validation.yml

```diff
@@ -44,4 +44,4 @@ jobs:
           kubeconform: latest
       - name: Check k8s manifests
         run: |
-          kustomize build $TUTOR_ROOT/env | kubeconform -strict -ignore-missing-schemas -kubernetes-version latest
+          kustomize build $TUTOR_ROOT/env | kubeconform -strict -ignore-missing-schemas -kubernetes-version master
```

Comment:
@marbonilla why is this change required??

Reply:
@jfavellar90  This change fixes the `Check k8s manifests` workflow so that it does not fail, however it is now in separate PRs